src/controllers/classesController.js

```diff
@@ -37,21 +37,30 @@ const displaySearchClasses = async (req, res) => {
           { description: searchRegex },
         ],
       };
-    }
-
-    const classes = await Class.find(query)
-      .skip(skip)
-      .limit(limit)
-      .sort({ [sortBy]: sortOrder });
-
-    const total = await Class.countDocuments(query);
 
-    res.status(StatusCodes.OK).json({
-      classes,
-      total,
-      totalPages: Math.ceil(total / limit),
-      currentPage: page,
-    });
+      const classes = await Class.find(query)
+        .skip(skip)
+        .limit(limit)
+        .sort({ [sortBy]: sortOrder });
+
+      const total = await Class.countDocuments(query);
+
+      res.status(StatusCodes.OK).json({
+        classes,
+        total,
+        totalPages: Math.ceil(total / limit),
+        currentPage: page,
+      });
+    } else {
+      const classes = await Class.find().sort({ [sortBy]: sortOrder });
```

Comment:
is this intentional change of logic - to show all classes on the one page if no search query?

Reply:
Yes, but I planned to rewrite displaySearchClasses in a separate PR to improve our search functionality.